frontend/app/src/components/helper/AssetDetailsBase.vue

```diff
@@ -4,7 +4,7 @@
     :class="opensDetails ? 'asset-details-base--link' : null"
     :dense="dense"
     :title="asset.isCustomAsset ? name : symbol"
-    :subtitle="asset.isCustomAsset ? asset.customAssetType : symbol"
+    :subtitle="asset.isCustomAsset ? asset.customAssetType : name"
```

Comment:
Not important right now but the original reason AssetDetailsBase/AssetDetails were different has been removed, we should evaluate if there is still any need for two different components and if there is not we should merge them into one.